mortgageAPI.yaml

```diff
@@ -2147,6 +2147,11 @@ components:
               $ref: '#/components/schemas/Date'
         purchaseDate:
           $ref: '#/components/schemas/Date'
+        estimationId
+          type: string
+          pattern: '^[a-zA-Z0-9]{6,10}$'
```

Comment:
Isn't this too specific? What if other tools are generating other ID formats? I'd suggest to keep the format completely open here (i.e., accept any string)